### DISCUSSION TITLE
How to ensure the security of chia wallet?

### DISCUSSION THREAD

**Discussion by jimuyouyou** [guydavis/machinaris#351] [General]
I noticed that chia, flax, hddcoin and nchain are using the same mnemonic words/keys for the farming. Let's say there will be a new fork name chia-thief, which maliciously capture/record the mnemonic words/keys, then they will be able to steal all other forks coins. Any idea on making it more safer? The only way I can think out is adding a periodically automatic transfer command to the machinaris docker environment, which will transfer the coins into another cold wallet immediately. 

I am new to machinaris project. Please correct me if my understanding is wrong.

**Comment by guydavis** [comment]
Good question.  Please see the wiki about [Cold Wallets](https://github.com/guydavis/machinaris/wiki/Keys#using-a-cold-wallet) usage generically with Chia.  As Machinaris uses the same Chia CLI binaries and configs behind the scenes the same approach works.
